input/examples/StructureDefinition-module-definition-library-cqfm.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Why is this file added?

Reply:
I have no clue.  It wasn't anything I did intentionally.  I have no clue.  Wasn't anything I intentionally added.  But I do have a tendency to rely on the `_genonce.sh` tool to re-build the IG, and I'm fairly sure that I ran `_updatePublisher.sh` sometime in the past month.  Maybe an update to dependencies caused it to get compiled or fetched from elsewhere?  I think this entry was auto-added.